_posts/2017-07-25-Creating-reactive-calendar-in-angular4.md

```diff
@@ -8,56 +8,56 @@ comments: true
+The user can interact with the following UI elements:
 
-- **Next button:** Allows the user to go to the next day in day-mode, week in week-mode, etc...
-- **Previous button:** Allows the user to go to the previous day in day-mode, week in week-mode, etc...
-- **Day, week, month buttons:** Allows the user to switch between the different viewmodes
-- **Searchterm input:** Allows the user to filter the appointments on the fly
+- **Next button:** Allows the user to go to the next day in day mode, week in week mode, etc.
+- **Previous button:** Allows the user to go to the previous day in day mode, week in week mode, etc.
+- **Day, week, month buttons:** Allows the user to switch between the different view modes
+- **Search term input:** Allows the user to filter the appointments on the fly
 - **Plus-buttons in the grid:** Allows the user to create new appointments
-- **Trashcan-buttons in the grid:** Allows the user to remove appointments
+- **Trashcan buttons in the grid:** Allows the user to remove appointments
 - **Description inputs:** Allows the user to update the description of an appointment
 
-I decided to use firebase as a backend and because of that, our application will be realtime and offline first by default!
+I decided to use Firebase as a backend and because of that, our application will be realtime and offline first by default!
 
-**Note:** One small issue, I've been a bit lazy so we can only create lunch-appointments =) But hey! Consider it some homework.
+**Note:** One small issue, I've been a bit lazy so we can only create lunch appointments. =) But hey! Consider it some homework.
 
 
-## Setting up the project
+## Setting Up the Project
 
-I've created the git branch **initial** to get us started. It contains the default logic/components, setup and styles. There is no reactive code written yet, just plain angular code. The goal is to write the reactive part ourselves.
+I've created the git branch **initial** to get us started. It contains the default logic/components, setup, and styles. There is no reactive code written yet, just plain Angular code. The goal is to write the reactive part ourselves.
 
 
-### The component tree
+### The Component Tree
 
 ![The component tree](https://raw.githubusercontent.com/brechtbilliet/brechtbilliet.github.io/master/_posts/reactivecalendar/reactivecalendar3.png)
 The dumb components (blue) are already implemented. The ```app-root``` (orange) is the one and only smart component in the application and the only place where we will write code.
 
 If you don't know the difference between smart and dumb components, [read this first](http://blog.brecht.io/components-demystified/#smart-vs-dumb-components).
 
 
-### Installing the project locally
-First of all, we have to clone the project locally, and checkout the **initial** branch. This branch already contains all the uninteresting parts that don't have anything to do with this article. 
+### Installing the Project Locally
```

Comment:
I have used the AP style of capitalization in titles, which is the style most widely used online. Only minor words such as "and, or, in" aren't capitalized in that style.